apluslms-yamlidator/apluslms_yamlidator/utils/error_render.py

```diff
@@ -0,0 +1,22 @@
+def print_lc(file_, line, col, num_lines=5, print_file=True):
```

Comment:
As the output is list of strings (or if it was just a string), then a name `render` is more suited. `print` indicates the funcion doesn't output anything.

Reply:
Or it could be `format` similar to functions in traceback module.